### PR TITLE
Limiters: use q_min/q_max instead of Q_min/Q_max

### PR DESCRIPTION
**Pull request by valeriabarra**
This PR addresses #687 . 

It modifies the limiters function and existing examples to use the q_min/q_max instead of Q_min/Q_max. As suspected, since we were always using a ρ starting = 1, we don't notice any difference up to round off precision.

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

**Comment by valeriabarra** [comment]
bors try

**Comment by valeriabarra** [comment]
bors try

**Comment by valeriabarra** [comment]
bors try

**Review by simonbyrne — APPROVED** [review]

**Comment by valeriabarra** [comment]
bors try

File: examples/plane/limiters_advection.jl [hunk 1]
```diff
@@ -189,33 +189,33 @@ for (k, ne) in enumerate(ne_seq)
...
+                    neigh_elems_q_max[i] = -Inf
                 else
-                    neigh_elems_Q_min[i] =
-                        Fields.minimum(Fields.slab(y.ρq, neigh_elems[i]))
-                    neigh_elems_Q_max[i] =
-                        Fields.maximum(Fields.slab(y.ρq, neigh_elems[i]))
+                    neigh_elems_q_min[i] =
+                        Fields.minimum(Fields.slab(y.ρq ./ y.ρ, neigh_elems[i]))
+                    neigh_elems_q_max[i] =
+                        Fields.maximum(Fields.slab(y.ρq ./ y.ρ, neigh_elems[i]))
```
Comment (simonbyrne) [inline examples/plane/limiters_advection.jl:214]: Looking at this more closely, should this be:
```suggestion
                    neigh_elems_q_min[e] =
                        Fields.minimum(Fields.slab(y.ρq ./ y.ρ, neigh_elems[i]))
                    neigh_elems_q_max[e] =
                        Fields.maximum(Fields.slab(y.ρq ./ y.ρ, neigh_elems[i]))
```
?

Comment (valeriabarra) [reply examples/plane/limiters_advection.jl:214]: Thanks for this review. I am puzzled to understand what is going on here in this PR now with CI. It used to work fine before being rebased onto `main` after the confusion about this file being accidentally edited/reverted by #689 . Could you please shed some light on what is going on? It seems like it just stalls after the `@ info` print-outs

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -293,7 +293,7 @@ steps:
       queue: central
       slurm_ntasks: 1
 
-  - label: ":computer: Advection CG plane limiter cosine bells"
+  - label: ":computer: Plane limiters advection cosine bells"
```
Comment (simonbyrne) [inline .buildkite/pipeline.yml:296]: I don't have a problem with renaming these, but I would prefer it to be a different PR

Comment (valeriabarra) [reply .buildkite/pipeline.yml]: Ok, sure. I opened #700 and dropped the commit here.

**Comment by valeriabarra** [comment]
bors try

**Comment by valeriabarra** [comment]
Although this PR did not change the resolution of the integration tests, it seems like bors times out now on these tests (?) I don't understand why. They work locally fine when tested on my machine. 
@simonbyrne do you understand what is going on here with CI?

File: examples/plane/limiters_advection.jl [hunk 3]
```diff
@@ -189,33 +189,33 @@ for (k, ne) in enumerate(ne_seq)
...
-                    neigh_elems_Q_max[i] = -Inf
+                    neigh_elems_q_min[i] = +Inf
+                    neigh_elems_q_max[i] = -Inf
                 else
-                    neigh_elems_Q_min[i] =
-                        Fields.minimum(Fields.slab(y.ρq, neigh_elems[i]))
-                    neigh_elems_Q_max[i] =
-                        Fields.maximum(Fields.slab(y.ρq, neigh_elems[i]))
+                    neigh_elems_q_min[i] =
+                        Fields.minimum(Fields.slab(y.ρq ./ y.ρ, neigh_elems[i]))
```
Comment (simonbyrne) [inline examples/plane/limiters_advection.jl:212]: You're doing `y.ρq ./ y.ρ` at every iteration of the loop

Comment (valeriabarra) [reply examples/plane/limiters_advection.jl:212]: What would you suggest we should do? We need this on each element slab

Comment (valeriabarra) [reply examples/plane/limiters_advection.jl:212]: One thing I can think of is returning `q` only instead of `ρq` from the limiters routine. But that will require some more changes

**Comment by valeriabarra** [comment]
bors try

**Comment by charleskawczynski** [comment]
It looks like this is significantly increasing the CI time.

**Comment by valeriabarra** [comment]
> It looks like this is significantly increasing the CI time.

It was, but actually with the latest edits, it decreased it significantly

**Comment by valeriabarra** [comment]
But if that's still too slow, I will reduce the number of outputs produced.

**Comment by valeriabarra** [comment]
bors try

**Comment by valeriabarra** [comment]
bors try-

**Comment by valeriabarra** [comment]
bors try
